unit_test/entities/test_esdl_object.py

```diff
@@ -90,8 +90,8 @@ def test_get_connected_assets(self):
         connected_assets2 = self.esdl_object.get_connected_assets(asset.id, asset.port[0].id)  # act
 
         # Assert
-        self.assertEqual(connected_assets1, test_list1)
-        self.assertEqual(connected_assets2, test_list2)
+        self.assertEqual(connected_assets1, test_list2)
```

Comment:
Why? This seems odd that you need to change the connection test when you are only changing the priority?

Reply:
This is connected to changing the port order in the esdl.